src/main/java/bedwars/Main.java

```diff
@@ -242,6 +245,8 @@ private void endGame() {
 			p.getInventory().clear();
 		}
 
+        playerUpgrades.resetUpgrades();
```

Comment:
I'll put this in startgame as well because sometimes to debug I reload the plugin before the game finishes

Reply:
sounds good